IdentityCore/src/oauth2/MSIDWebviewFactory.m

```diff
@@ -181,6 +181,14 @@ @implementation MSIDWebviewFactory
         result[MSID_OAUTH2_CODE_CHALLENGE] = pkce.codeChallenge;
         result[MSID_OAUTH2_CODE_CHALLENGE_METHOD] = pkce.codeChallengeMethod;
     }
+    
+    // Nested auth protocol
+    if ([parameters isNestedAuthProtocol])
+    {
+        MSID_LOG_WITH_CTX(MSIDLogLevelInfo, nil, @"Nested auth protocol - Adding broker client id & redirect uri to webview");
+        result[MSID_NESTED_AUTH_BROKER_CLIENT_ID] = parameters.nestedClientId;
```

Comment:
Use consistent naming, should be called nestedBrokerClientId inside request parameters as well

Reply:
Updated